config/initializers/csv_safe.rb

```diff
@@ -0,0 +1,12 @@
+require "csv-safe"
+
+class CSVSafe
+  def sanitize_field(field)
+    # we may represent monetary numerics as strings to format them, so we don't want CSVSafe to prefix them
+    if field.nil? || field.is_a?(Numeric) || field.is_a?(String) && Regexp.new(/\A[-\d.,]+\z/).match?(field)
```

Comment:
Could we label the regex in a variable to clarify what it's matching (more specifically than in the comment above)?

Reply:
Is it better now that I've extracted the conditional into its own method?

Reply:
Looks good!